### ISSUE TITLE
Collagenoma

### ISSUE THREAD

**Issue by jagcgc** [obophenotype/human-phenotype-ontology#10245]
**Preferred term label:**
Collagenoma

**Synonyms**
None

**Definition (free text, please give PubMed ID)**
PMID: [29445568](https://pubmed.ncbi.nlm.nih.gov/29445568)
A rare hamartomatous malformation characterized by the predominant proliferation of normal collagen fibers and normal, decreased, or increased elastic fibers. Collagenomas may be solitary or multiple, sporadic, acquired or inherited.

**Parent term (use hpo.jax.org/app)**
Connective tissue nevi [HP:0100898](https://hpo.jax.org/app/browse/term/HP:0100898)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
[OMIM:115250](https://hpo.jax.org/app/browse/disease/OMIM:115250) Collagenoma, familial cutaneous

**Your nano-attribution (ORCID)**
0000-0001-8481-3559

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000022
- label: Collagenoma
- parents: HP:0100898
- pmid: PMID:29445568
- def: A rare hamartomatous malformation characterized by the predominant proliferation of normal collagen fibers and normal, decreased, or increased elastic fibers. Collagenomas may be solitary or multiple, sporadic, acquired or inherited.
- comment: 
- orcid: orcid:0000-0001-8481-3559
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10245
- synonyms:
